skaffold.yaml

```diff
@@ -1620,13 +1620,6 @@ profiles:
           - bash
           - -ec
           - KUBECONFIG=$GARDENER_LOCAL_KUBECONFIG hack/usage/wait-for.sh seed "$SEED_NAME" GardenletReady SeedSystemComponentsHealthy ExtensionsReady
-  - name: dual
-    activation:
-      - env: IPFAMILY=dual
-    patches:
-      - op: add
-        path: /deploy/helm/releases/0/valuesFiles/-
-        value: example/gardener-local/gardenlet/values-dual.yaml
```

Comment:
When we drop this we won't have dual-stack seed configuration for the gardenlet for `make kind-up gardener-up IPFAMILY=dual`. 
Is this not supported anymore?

Reply:
Yes, it's not supported anymore. We are moving all setups to `gardener-operator`-based setups (intention of this PR, actually 😅). The Helm chart-based `gardener-up` rule will soon disappear (or at least work with `gardener-operator` exclusively).